integration-tests/src/bin/write.rs

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use libc::write;
+
+fn main() {
+    let buf = "hello world\n";
+    unsafe {
+        write(libc::STDOUT_FILENO, buf.as_ptr() as *const _, buf.len());
+    }
+}
```

Comment:
Don't forget to add a second test which does exactly the same thing on `libc::STDERR_FILENO`.